### PR TITLE
Fixed memory parameters and plot output in wdl

### PR DESCRIPTION
**Pull request by asmirnov239**
This pull request contains two fixes (split into 2 commits):
1) Exposed memory and other CalculateTargetCovererage parameters
2) Adding corresponding output for the plotting task in wdl, which required providing an option in plotting CLI to name the file.

### CODE REVIEW

**Comment by asmirnov239** [comment]
@samuelklee Could you please review this?

**Comment by samuelklee** [comment]
Will do!  Note that you may have to resolve some conflicts if @LeeTL1220 merges #650 first.

File: scripts/acnv_cromwell_tests/case_gatk_acnv_workflow_exome.json [hunk 1]
```diff
@@ -1,20 +1,20 @@
...
   "case_gatk_acnv_workflow.input_bam_list": "/home/travis/build/broadinstitute/gatk-protected/scripts/acnv_cromwell_tests/case_input_exome.tsv",
-  "case_gatk_acnv_workflow.CalculateTargetCoverage.keepduplicatereads": "true",
+  "case_gatk_acnv_workflow.keep_duplicate_reads": "true",
   "case_gatk_acnv_workflow.PoN": "/home/travis/build/broadinstitute/gatk-protected/src/test/resources/large/ACNV_pipeline_test_files/exome_test.pon",
-  "case_gatk_acnv_workflow.CalculateTargetCoverage.transform": "PCOV",
+  "case_gatk_acnv_workflow.transform": "PCOV",
   "case_gatk_acnv_workflow.ref_fasta": "/home/travis/build/broadinstitute/gatk-protected/src/test/resources/large/ACNV_pipeline_test_files/human_g1k_v37.chr-20.truncated.fasta",
-  "case_gatk_acnv_workflow.NormalizeSomaticReadCounts.mem": "4",
-  "case_gatk_acnv_workflow.PadTargets.pd": "250",
+  "case_gatk_acnv_workflow.norm_somatic_read_count_memory": "4",
```
Comment (samuelklee) [inline scripts/acnv_cromwell_tests/case_gatk_acnv_workflow_exome.json]: Use `normalize_somatic_read_count_memory`, just so there's no confusion between normal (as in normal/tumor) and normalize.

File: scripts/acnv_cromwell_tests/case_gatk_acnv_workflow_exome.json [hunk 2]
```diff
@@ -1,20 +1,20 @@
...
+  "case_gatk_acnv_workflow.grouping": "SAMPLE",
   "case_gatk_acnv_workflow.common_snp_list": "/home/travis/build/broadinstitute/gatk-protected/src/test/resources/large/ACNV_pipeline_test_files/common_snps_sample-chr20.interval_list",
   "case_gatk_acnv_workflow.ref_fasta_fai": "/home/travis/build/broadinstitute/gatk-protected/src/test/resources/large/ACNV_pipeline_test_files/human_g1k_v37.chr-20.truncated.fasta.fai",
   "case_gatk_acnv_workflow.target_file": "/home/travis/build/broadinstitute/gatk-protected/src/test/resources/large/ACNV_pipeline_test_files/ice_targets_sample-chr20.tsv",
   "case_gatk_acnv_workflow.gatk_jar": "/home/travis/build/broadinstitute/gatk-protected/build/libs/gatk-protected.jar",
-  "case_gatk_acnv_workflow.PadTargets.mem": "1",
-  "case_gatk_acnv_workflow.CalculateTargetCoverage.disable_all_read_filters": "false",
-  "case_gatk_acnv_workflow.CalculateTargetCoverage.mem": "4",
+  "case_gatk_acnv_workflow.disable_all_read_filters": "false",
+  "case_gatk_acnv_workflow.calc_target_cov_memory": "4",
```
Comment (samuelklee) [inline scripts/acnv_cromwell_tests/case_gatk_acnv_workflow_exome.json]: I think it's fine to be a little more verbose here and use `calculate_target_coverage_memory`.

File: scripts/acnv_cromwell_tests/case_gatk_acnv_workflow_wgs.json [hunk 3]
```diff
@@ -1,20 +1,20 @@
...
+  "case_gatk_acnv_workflow.grouping": "SAMPLE",
   "case_gatk_acnv_workflow.common_snp_list": "/home/travis/build/broadinstitute/gatk-protected/src/test/resources/large/ACNV_pipeline_test_files/common_snps_sample-chr20.interval_list",
   "case_gatk_acnv_workflow.ref_fasta_fai": "/home/travis/build/broadinstitute/gatk-protected/src/test/resources/large/ACNV_pipeline_test_files/human_g1k_v37.chr-20.truncated.fasta.fai",
   "case_gatk_acnv_workflow.target_file": "/home/travis/build/broadinstitute/gatk-protected/src/test/resources/large/ACNV_pipeline_test_files/ice_targets_sample-chr20.tsv",
   "case_gatk_acnv_workflow.gatk_jar": "/home/travis/build/broadinstitute/gatk-protected/build/libs/gatk-protected.jar",
-  "case_gatk_acnv_workflow.PadTargets.mem": "1",
-  "case_gatk_acnv_workflow.CalculateTargetCoverage.disable_all_read_filters": "false",
-  "case_gatk_acnv_workflow.CalculateTargetCoverage.mem": "4",
+  "case_gatk_acnv_workflow.disable_all_read_filters": "false",
+  "case_gatk_acnv_workflow.calc_target_cov_memory": "4",
```
Comment (samuelklee) [inline scripts/acnv_cromwell_tests/case_gatk_acnv_workflow_wgs.json]: See above.

File: scripts/acnv_cromwell_tests/case_gatk_acnv_workflow_wgs.json [hunk 4]
```diff
@@ -1,20 +1,20 @@
...
   "case_gatk_acnv_workflow.input_bam_list": "/home/travis/build/broadinstitute/gatk-protected/scripts/acnv_cromwell_tests/case_input_wgs.tsv",
-  "case_gatk_acnv_workflow.CalculateTargetCoverage.keepduplicatereads": "true",
+  "case_gatk_acnv_workflow.keep_duplicate_reads": "true",
   "case_gatk_acnv_workflow.PoN": "/home/travis/build/broadinstitute/gatk-protected/src/test/resources/large/ACNV_pipeline_test_files/wgs_test.pon",
-  "case_gatk_acnv_workflow.CalculateTargetCoverage.transform": "PCOV",
+  "case_gatk_acnv_workflow.transform": "PCOV",
   "case_gatk_acnv_workflow.ref_fasta": "/home/travis/build/broadinstitute/gatk-protected/src/test/resources/large/ACNV_pipeline_test_files/human_g1k_v37.chr-20.truncated.fasta",
-  "case_gatk_acnv_workflow.NormalizeSomaticReadCounts.mem": "4",
-  "case_gatk_acnv_workflow.PadTargets.pd": "250",
+  "case_gatk_acnv_workflow.norm_somatic_read_count_memory": "4",
```
Comment (samuelklee) [inline scripts/acnv_cromwell_tests/case_gatk_acnv_workflow_wgs.json]: See above.

File: scripts/acnv_cromwell_tests/pon_gatk_workflow_exome.json [hunk 5]
```diff
@@ -1,7 +1,14 @@
 {
+  "case_gatk_acnv_workflow.PadTargets.padding": "250",
   "pon_gatk_workflow.disable_sequence_dictionary_validation": "true",
-  "pon_gatk_workflow.isWGS": "false",
+  "pon_gatk_workflow.disable_all_read_filters": "false",
+  "pon_gatk_workflow.keep_duplicate_reads": "true",
+  "pon_gatk_workflow.transform": "PCOV", 
+  "pon_gatk_workflow.grouping": "SAMPLE",
+  "pon_gatk_workflow.calc_target_cov_memory": "4",
```
Comment (samuelklee) [inline scripts/acnv_cromwell_tests/pon_gatk_workflow_exome.json]: See above.

File: scripts/acnv_cromwell_tests/pon_gatk_workflow_wgs.json [hunk 6]
```diff
@@ -1,7 +1,14 @@
 {
+  "case_gatk_acnv_workflow.PadTargets.padding": "250",
   "pon_gatk_workflow.disable_sequence_dictionary_validation": "true",
-  "pon_gatk_workflow.isWGS": "true",
+  "pon_gatk_workflow.disable_all_read_filters": "false",
+  "pon_gatk_workflow.keep_duplicate_reads": "true",
+  "pon_gatk_workflow.transform": "PCOV", 
+  "pon_gatk_workflow.grouping": "SAMPLE",
+  "pon_gatk_workflow.calc_target_cov_memory": "4",
```
Comment (samuelklee) [inline scripts/acnv_cromwell_tests/pon_gatk_workflow_wgs.json]: See above.

File: scripts/wdl/case_gatk_acnv_workflow.wdl [hunk 7]
```diff
@@ -52,14 +52,25 @@ workflow case_gatk_acnv_workflow {
...
+    # Workflow output directories and other options
     String plots_dir
     String call_cnloh_dir
-    Boolean disable_sequence_dictionary_validation
     Boolean enable_gc_correction
     Boolean isWGS
     Int wgsBinSize
 
+    # Java maximum memory options
+    Int calc_target_cov_memory
```
Comment (samuelklee) [inline scripts/wdl/case_gatk_acnv_workflow.wdl]: See above.

File: scripts/wdl/pon_gatk_workflow.wdl [hunk 8]
```diff
@@ -33,17 +33,27 @@ workflow pon_gatk_workflow {
...
     Int wgsBinSize
+    Int max_open_files
+    String combined_entity_id = "combined_read_counts"
 
     # PoN name
     String pon_entity_id
-    
+
+    # Java maximum memory options
+    Int calc_target_cov_memory
```
Comment (samuelklee) [inline scripts/wdl/pon_gatk_workflow.wdl]: See above.

**Comment by samuelklee** [comment]
Looks good!  Just a couple of minor suggestions for making the parameter names more verbose.  Looks like @LeeTL1220 merged already, so you'll have to resolve conflicts in the json templates.

**Pull request merged**